<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #896e69</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f9f9ff;</string>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f9f9ff;</string>
    </property>
    <property name="text">
     <string>Использование дополнительного
 алгоритма сравнения хэшей изображения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f9f9ff;</string>
    </property>
    <property name="text">
     <string>Использования дополнительного
 алгоритма сравнения изображний</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f9f9ff;</string>
    </property>
    <property name="text">
     <string>Использование видеокарты
(если она присутствует)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffa577;color: #f9f9ff; border-color: #f9f9ff;border-width: 1px;border-style: solid;border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Сохранить настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffa577;color: #f9f9ff; border-color: #f9f9ff;border-width: 1px;border-style: solid;border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;</string>
    </property>
    <item>
     <property name="text">
      <string>Нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Да</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;</string>
    </property>
    <item>
     <property name="text">
      <string>Нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Да</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;</string>
    </property>
    <item>
     <property name="text">
      <string>Нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Да</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
